source/CCustomOpcodeSystem.cpp

```diff
@@ -1350,6 +1350,10 @@ namespace CLEO
 		case 2:
 			GetInstance().CodeInjector.MemoryWrite(Address, (WORD)value, vp);
 			break;
+		case 3:
+			GetInstance().CodeInjector.MemoryWrite(Address, (WORD)value, vp);
+			GetInstance().CodeInjector.MemoryWrite(Address + 2, *((BYTE*)&value + 2), vp, 1); // remaining 1 byte
+			break;

```

Comment:
case 3 is redundant, as default already covers it

Reply:
Used underneath memcpy acts like memfill when provided size 3. Writes 3 times same byte.

Reply:
this is a breaking change then. someone could use it, for example to nop 3 bytes.
```
write_memory 0xabcdef size 3 value 0x90 vp 1
```

we should add a cs4 compat mode here